challenges/classes.js

```diff
@@ -22,34 +22,55 @@ CuboidMaker.prototype.volume = function () {
 
   Formula for cuboid surface area of a cube: 2 * (length * width + length * height + width * height)
 */
-CuboidMaker.prototype.surfaceArea = function () {
+/*CuboidMaker.prototype.surfaceArea = function () {
   return 2 * (this.length * this.width + this.length * this.height + this.width + this.height);
 };
 
 /* == Step 4: Create a new object that uses CuboidMaker ==
   Create a cuboid object that uses the new keyword to use our CuboidMaker constructor
   Add properties and values of length: 4, width: 5, and height: 5 to cuboid.   
 */
-const cuboid = new CuboidMaker ({
+/*const cuboid = new CuboidMaker ({
   length: 4,
   width: 5,
   height: 5
 });
+*/
 
+class CuboidMaker {
+  constructor (attributes) {
+    this.length = attributes.length;
+    this.width = attributes.width;
+    this.height = attributes.height;
+  }
 
+  volume() {
+    return this.length * this.width * this.height;
+  }
+  
+  surfaceArea() {
+    return 2 * (this.length * this.width + this.length * this.height + this.width + this.height);
```

Comment:
Here again, it needs to be multiplication instead of addition. Just a small change,